src/power-flow-card.ts

```diff
@@ -76,9 +78,13 @@ export class PowerFlowCard extends LitElement {
   };
 
   private displayValue = (value: number) =>
-    value >= this._config!.watt_threshold
-      ? `${round(value / 1000, this._config!.kw_decimals)} kW`
-      : `${round(value, 1)} W`;
+    if (value >= this._config!.watt_threshold)
+      `${round(value / 1000, this._config!.kw_decimals)} kW`;
+    else {
+      if (this._config!.w_decimals > 0)
+        `${round(value, this._config!.w_decimals)} W`;
+      else `${round(value)} W`;
+    };
```

Comment:
An if statement won't work in an arrow function.  Also we don't need to check if it's greater than zero, as someone may prefer to round to no decimals.  Just changing line 81 to ``: `${round(value, this._config!.w_decimals)} W`; `` should be sufficient.

Reply:
I disagree, I would prefer a clean _XXXW_ reading than _XXX.0W_. The nested if is meant to avoid that.